lessons/Assignment-3-Ruby-Part-1.md

```diff
@@ -11,18 +11,21 @@ Create second.rb.  It should do the following:
 - Put the keys for the hash to the console, also using puts. Hint: The Hash class has a method you can call.  Note that you get an array back.
 - Put the values for the hash to the console, also using puts.
 - Change the first name, last name, and city as stored in the hash, so that each of these is capitalized.  Hint: You can use the capitalize method of the string class.  So you would have:
-    ```
+
+    ```ruby
     person[:first_name] = person[:first_name].capitalize
     ```
+
 - Change the state as stored in the person hash to upper case.
 - Put the hash to the console again.
 
-Functions follow the same naming convention as variables, except that a function might end with a "?", which means it returns true or false; or a function might end with a "!", which means that it changes the passed in value (often an array or hash), as well as returning the chamged value; or a function might end with a "=" which means that it's a setter.  We'll get to setters when we get to classes. 
+Methods follow the same naming convention as variables, except that a method might end with a `?`, which means it returns true or false (a "boolean"); or a method might end with a `!`, which means that it changes the passed in value (often an array or hash), as well as returning the changed value; or a method might also end with a `=` which means that it's a setter.  We'll get to setters when we get to classes.
```

Comment:
"Functions" are not really a thing in Ruby; even at the top level, everything is part of an object. In irb:

```ruby
method(:puts)
=> #<Method: Object(Kernel)#puts(*)>

self
=> main

self.class
=> Object

def foo; end
method(:foo)
=> => #<Method: Object#foo() (irb):4>
```